css/styles.css

```diff
@@ -0,0 +1,93 @@
+/* most of your CSS that is not related to layout or resetting styles here */
+.button {
+  display: block;
+  margin: 0 auto;
+  color: white;
+  background-color: #293184;
+  border-radius: 4px;
+  padding-left: 1.7rem;
+  padding-right: 1.7rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  cursor: pointer;
+}
+
+header {
+  color: white;
+  background-color: #293184;
+  font-size: large;
```

Comment:
I don't recommend using the built in sizes for your `font-size`. What is `large`? What is `medium`? It would be better to have full control over what we want that size to be.